phpunit.xml.dist

```diff
@@ -19,6 +19,7 @@
   <php>
     <ini name="error_reporting" value="-1" />
     <ini name="xdebug.overload_var_dump" value="0" />
+    <server name="SYMFONY_PHPUNIT_VERSION" value="7.5" />
```

Comment:
I don't know if this is the convention outside of Symfony, but with this, phpunit 8.x was being pulled down.